book/website/reproducible-research/licensing/licensiing-hardware.md

```diff
@@ -0,0 +1,193 @@
+Licensing is an important aspect of sharing/publishing open source projects as it provides clarity for anyone looking to reuse your open source project.
+Without licenses in place, anyone who wants to reuse it, is left with legal ambiguity as to the status of using your intellectual property.
+While we are looking to focus on open source hardware licenses, this should be considered within the broader open source movement. 
+The open source licensing of hardware came out of open source software licenses and, in fact, several open source software licenses are widely accepted within the open hardware community. 
+
+```{warning}
+It is important to consider that copyright, licenses, and patents are all legal concepts.
+As such, they are subject to what the law prescribe.
+And laws can change.
+This change is not only that one of the laws is changing over time, but it changes over space: depending on jurisdiction and enforcement.
+Simply put, different countries have different laws, and follow different procedures with regard to enforcing them.
+The content provided here is broadly based on American and European law and legal traditions.
+It might not be applicable - might even be contra indicated - or relevant in your particular context.
+While things like the Berne Convention have sought to harmonize copyright enforcement, the real world is a messy place.
+
+Please also note that the information provided here was not written by a lawyer.
+So do not take the descriptions provided or viewpoints shared as legal advice, they are not that.
+This document is not intended to be used in that manner.
+Consult a legal expert to provide actual legal advice for your case.  
+```
+
+## Content
+To understand what licenses do and how to apply them, it is first important to be familiar with a number of terms.
+Often terms like copyright, licenses, patents, and trademarks are used interchangeably (sometimes incorrectly).
+This may lead to confusion as to what exactly is being discussed and what the implications are.
+Familiarise yourself with the following terms and follow the links to understand more about them.
+
+
+```{admonition} Info
+**[Copyright](https://europa.eu/youreurope/business/running-business/intellectual-property/copyright/index_en.htm#shortcut-1/):** “When you create an original literary, scientific and artistic work, such as poems, articles, films, songs or sculptures, you are protected by copyright.
+Nobody apart from you has the right to make the work public or reproduce it.
+” And, “If you create literary, scientific and artistic work, you automatically have copyright protection, which starts from the moment you create your work, so you don't need to go through any formal application process.”
+
+**[License](https://www.oshwa.org/faq/#what-is-a-license/):** “ Licensing is a way to give people rights that they wouldn’t otherwise have to use your intellectual property (IP) and, in return, to place restrictions on how they exercise those rights.”
```

Comment:
```suggestion
**[License](https://www.oshwa.org/faq/#what-is-a-license/):** “Licensing is a way to give people rights that they wouldn’t otherwise have to use your intellectual property (IP) and, in return, to place restrictions on how they exercise those rights.”
```